README.md

```diff
@@ -56,6 +56,17 @@ The Ibex user manual can be
 [read online at ReadTheDocs](https://ibex-core.readthedocs.io/en/latest/). It is also contained in
 the `doc` folder of this repository.
 
+## Examples
+
+The Ibex repository includes [Simple System](examples/simple_system/README.md).
+This is an intentionally simple integration of Ibex with a basic system that targets simulation.
+It is intended to provide an easy way to get bare metal binaries running on Ibex in simulation.
+
+A more complete example can be found in the [Ibex Super System repository](https://github.com/GregAC/ibex_super_system).
+In particular it includes a integration of the [PULP RISC-V debug module](https://github.com/pulp-platform/riscv-dbg).
+It targets the [Arty A7 FPGA board from Digilent](https://digilent.com/shop/arty-a7-artix-7-fpga-development-board/) and supports debugging via OpenOCD and GDB over USB (no external JTAG probe required).
+The Ibex Super System is written by Ibex contributor @GregAC but is not an official part of Ibex, nor officially supported by lowRISC.
```

Comment:
I don't think that the "@NAME" form gets automatically linked. I'd probably either suggest inserting a manual link or dropping the name entirely.

Reply:
Yeah it doesn't. Felt linking my own github profile in the README was a bit much so went for this. I think I'll just drop the name.